### PR TITLE
modify redfish discovery to optionally catalog EMC elements

### PR DESCRIPTION
**Pull request by zyoung51**
- Modify the redfish discovery graph to also catalog EMC
  elements on freshly added chassis/systems when auto
  catalog is enabled

### CODE REVIEW

**Comment by zyoung51** [comment]
@benbp Have you seen the heartbeat failure in task-runner before: https://travis-ci.org/RackHD/on-taskgraph/jobs/120125182 ?

**Comment by benbp** [comment]
@VulpesArtificem see the comment above ^^. @zyoung51 I bet it's an error with using timers in the test environment.

**Comment by benbp** [comment]
Although I would have thought the `.take(5)` call at https://github.com/RackHD/on-taskgraph/blob/master/spec/lib/task-runner-spec.js#L252 would completely remove the possibility of calling the stub 6 times??

File: lib/graphs/redfish-discovery-graph.js [hunk 1]
```diff
@@ -8,12 +8,42 @@ module.exports = {
     options: {
         defaults: {
             uri: null
+        },
+        'when-catalog-emc' : {
+            autoCatalogEmc: 'false',
```
Comment (yyscamper) [inline lib/graphs/redfish-discovery-graph.js]: The `Boolean` type should be better than `String`

**Comment by yyscamper** [comment]
+1

**Comment by zyoung51** [comment]
test this please

**Comment by tannoa2** [comment]
+1

**Pull request merged**